guides/plugins/plugins/storefront/using-a-modal-window.md

```diff
@@ -0,0 +1,368 @@
+
+{% page-ref page="./../plugin-base-guide.md" %}
+
+While this is not mandatory, having read the guide about adding custom JavaScript plugins beforehand might help you understand this guide a bit further:
+
+{% page-ref page="./add-custom-javascript.md" %}
+
+
+## Create a modal manually from the DOM using Boostrap
+
+The simples solution to create a modal is by using Boostrap. More info: [Modal Boostrap](https://getbootstrap.com/docs/4.0/components/modal/#live-demo)
+Here is a basic implementation as an example. We override the `base_main_inner` from the `@Storefront/storefront/page/content/index.html.twig` template to insert the modal specific DOM elements.
+
+{% code title="<plugin root>/src/Resources/views/storefront/page/content/index.html.twig" %}
+```text
+{% sw_extends '@Storefront/storefront/page/content/index.html.twig' %}
+
+{% block base_main_inner %}
+    <!-- Button trigger modal -->
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModal">
+        Launch demo modal
+    </button>
+
+    <!-- Modal -->
+    <div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel">Modal title</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <!-- insert your content here -->
+                    ...
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                    <button type="button" class="btn btn-primary">Save changes</button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    {{ parent() }}
+{% endblock %}
+```
+{% endcode %}
+
+## Create a modal using Ajax Modal Utility
+
+When setting a `data-url` in addition to `data-toggle="modal"` shopware automatically uses the `PseudoModalUtil` and the pseudo modal template from the `base.html.twig` to render a modal:
+
+{% code title="<plugin root>/src/Resources/views/storefront/page/content/index.html.twig" %}
+```text
+{% sw_extends '@Storefront/storefront/page/content/index.html.twig' %}
+
+{% block base_main_inner %}
+    <!-- This uses `AjaxModalExtensionUtil` -->
+    <button class="btn"
+            data-toggle="modal"
+            data-url="https://example.org/ajax-url">
+
+    {{ parent() }}
+{% endblock %}
+```
+{% endcode %}
+
+{% hint style="warning" %}
+This does not work when the trigger selector is being changed via JavaScript, e.g. because of an ajax call which replaces the content.
```

Comment:
You write ajax here in small letters and later in capital. I think you want to write it in capitals.